sklearn/compose/_column_transformer.py

```diff
@@ -55,7 +55,8 @@ class ColumnTransformer(_BaseComposition, TransformerMixin):
         name : string
             Like in Pipeline and FeatureUnion, this allows the transformer and
             its parameters to be set using ``set_params`` and searched in grid
-            search.
+            search. The name 'remainder' can not be used if the ``remainder``
```

Comment:
I think we should outlaw the name remainder in all cases. In fact, I think `_BaseComposition._validate_names` enforces this.